### PR TITLE
SET STCGRP= for ACF2 using zowe.yaml values

### LINKED ISSUE #3498: For ACF2 the STCGRP needs to be set on line 174 of the `zwe init certificate` generated JCL
**Describe the bug**
The command `zwe init certificate` is incomplete for ACF2 in the JCL it generates

**To Reproduce**
Set the `zowe.security.product` to be `ACF2.
Run `zwe init certificate -c zowe.yaml --security-dry-run` and look at the generated JCL, especially line 174

```
//*
//* ACF2 ONLY -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -
//*                     12345678
//         SET  STCGRP=
//*                     12345678
//*
//* end ACF2 ONLY -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -
```
**Expected behavior**
the value of of the `STCGRP=` should have been picked up from `zowe.security.product.groups.stc`.  

**Screenshots**
 
image

### PR DESCRIPTION
**Pull request by Joe-Winchester**
Please check if your PR fulfills the following requirements. This is simply a reminder of what we are going to look for before merging your PR. If you don't know all of this information when you create this PR, don't worry. You can edit this template as you're working on it.

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Necessary documentation (if appropriate) have been added / updated
- [ ] DCO signoffs have been added to all commits, including this PR

#### PR type
What type of changes does your PR introduce to Zowe? _Put an `x` in the box that applies to this PR. If you're unsure about any of them, don't hesitate to ask._ 
- [X] Bugfix 
- [ ] Feature 
- [ ] Other... Please describe:

#### Relevant issues

Fixes #3498 

#### Changes proposed in this PR

Add line 174 to ZWEKRING JCL created by `zwe init certificate` to include the STC group for ACF2.

#### Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

#### Does this PR do something the person installing Zowe should know about?

---
* Affected function: _general area of interest_ *

---
* Description: _1 line description_ *

---
* Part: _name of customizable file involved_  *

---
_multi-line description_

#### Is there a related doc issue or Pull Request? 

Doc issue/PR number:

### CODE REVIEW

File: bin/libs/certificate.sh [hunk 1]
```diff
@@ -883,9 +883,11 @@ EOF
     racf_connect2="s/^ \+LABEL[(]'certlabel'[)].*\$/            LABEL('${connect_label}') +/"
   fi
 
-  # used by ACF2
-  # TODO: how to pass this?
-  stc_group=
+  # used by ACF2  
+  stc_group=$(read_yaml "${ZWE_CLI_PARAMETER_CONFIG}" ".zowe.setup.security.groups.stc")
+  if [ -z "${security_groups_stc}" ]; then
```
Comment (MarkAckert) [inline bin/libs/certificate.sh]: should this be `if [-z "${stc_group}" ]; then` ?

**Comment by MarkAckert** [comment]
Thanks for the PR...there's one more instance of stc_group [here: keyring_run_zwenokyr_jcl](https://github.com/zowe/zowe-install-packaging/blob/15f9f44d76797d41ecdea46f92a6fb391ff55174/bin/libs/certificate.sh#L1016) , and the STCGRP is used in the ZWENOKYR JCL

File: bin/libs/certificate.sh (hunk 1 above)
Comment (Martin-Zeithaml) [inline bin/libs/certificate.sh]: Is this correct? The variable 1 line above is `stc_group` and then `security_groups_stc` is being checked if empty.
Seems to be [copy/paste](https://github.com/zowe/zowe-install-packaging/blob/d8d5e3f9799ba5704eb7355a5afe7e898fce5c37/bin/commands/init/security/index.sh#L41) bug.

Comment (Joe-Winchester) [reply bin/libs/certificate.sh]: Sweet !  Thanks for picking up my mistake.   My bad for not checking :-(
I'm still waiting for a build before I do a quick manual test just to eyeball the JCL that `zwe init certificate` creates, but as I don't have access to an ACF2 System I won't be able to live test it.

**Review by Martin-Zeithaml — APPROVED** [review]
Approved.

**Review by MarkAckert — APPROVED** [review]

**Pull request merged**